src/modules/LanItems.rb

```diff
@@ -2203,17 +2203,24 @@ def enableCurrentEditButton
 
     # Creates eth emulation for s390 devices
     #
-    # @param [Hash] an s390 device description as obtained from AY profile
-    def createS390Device(rule)
+    # @param [Hash] an s390 device description (e.g. as obtained from AY profile).
+    # if not present then initialization of S390 specific LanItems' variables is
```

Comment:
"Not present" is slightly ambiguous, I wouldn't know whether that means `empty?` or `nil?` -> "If empty then..."